### PR TITLE
Added Blueprint Subsystems (Authentication, Wallet, Indexer)

### PR DESCRIPTION
**Pull request by andygruening**
Please view the 'SequenceBlueprintExamples' asset inside the Demonstration folder on how you use the Subsystems inside Blueprints.

### CODE REVIEW

**Review by caballoninja — DISMISSED** [review]
Made some changes for visual studio compiling, but works great

**Review by colezemind — DISMISSED** [review]
Updates look great, no issues from my end

**Review by BellringerQuinn — APPROVED** [review]
Thanks Andy! The example BPs look great by the way!

File: Plugins/SequencePlugin/Source/SequencePlugin/Private/Integrators/SequenceSdkBP.cpp [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+#include "Engine/Engine.h"
+#include "Engine/GameInstance.h"
+#include "Util/Log.h"
+
+USequenceSdkBP::USequenceSdkBP() { }
+
+void USequenceSdkBP::SetupAsync()
+{
+	// TODO: Integrate async setup process like querying remote chain configs
```
[the remark this answers is not in the document]
Comment (andygruening) [reply Plugins/SequencePlugin/Source/SequencePlugin/Private/Integrators/SequenceSdkBP.cpp:13]: Its for a future ticket, I saw SequenceSdkBP not needed right now, but I left it there when we need it for the remote chain configs.

**Review by colezemind — DISMISSED** [review]
New additions run fine, thanks for updating the blue print examples too.

**Review by caballoninja — APPROVED** [review]
new commits are good

File: Plugins/SequencePlugin/Source/SequencePlugin/Public/Integrators/SequenceIndexerBP.h [hunk 2]
```diff
@@ -38,7 +38,7 @@ class SEQUENCEPLUGIN_API USequenceIndexerBP : public UGameInstanceSubsystem
 	FOnGetTransactionHistory TransactionHistoryReceived;
 
 	UFUNCTION(BlueprintCallable, Category="0xSequence SDK - Functions")
-	void SetChainId(int64 NewChainId);
+	void SetChain(const FString& ChainName);
```
[the remark this answers is not in the document]
Comment (andygruening) [reply Plugins/SequencePlugin/Source/SequencePlugin/Public/Integrators/SequenceIndexerBP.h]: Added

**Pull request merged**